mrp_subcontracting_picking/readme/HISTORY.rst

```diff
@@ -0,0 +1,4 @@
+14.0.1.0.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
No need of this file, but if put, the underlining should be of the same length of the contents

Reply:
Done